crates/autopilot/src/on_settlement_event_updater.rs

```diff
@@ -215,9 +166,10 @@ impl OnSettlementEventUpdater {
             effective_gas_price,
             surplus,
             fee,
-            prices,
         };
 
+        tracing::debug!("updating settlement details for tx {hash:?}: {update:?}");
```

Comment:
You usually get nicer (and easier grepable) log messages by following this pattern:
```suggestion
        tracing::debug!(?hash, ?update, "updating settlement details for tx");
```

Reply:
Fixed.